source/Cosmos.HAL2/Network/NetworkInit.cs

```diff
@@ -41,6 +41,26 @@ public static void Init()
 
                     #endregion
 
+                    #region RTL8168
+
+                    if (device.VendorID == 0x10EC && device.DeviceID == 0x8168)
+                    {
+
+                        Console.WriteLine("NIC IRQ: " + device.InterruptLine);
+
+                        var RTL8168Device = new RTL8168(device);
```

Comment:
How is the device accessible from outside the class? Again I dont like the Console.WriteLines, maybe make the Init(bool aShowOutput) have a toggle for console output?